Makefile

```diff
@@ -0,0 +1,43 @@
+# These variables follow the naming convention from the GNU Make documentation
+# but their defaults correspond to the rest of the code (note that changing
+# libexecdir here wouldn't affect the default path for the channel plugin
+# directory used by the dameon for example).
+#
+# https://www.gnu.org/software/make/manual/html_node/Directory-Variables.html
+prefix     = /usr
+sbindir    = $(prefix)/sbin
+libexecdir = $(prefix)/libexec
+datadir    = $(prefix)/share
+sysconfdir = /etc
+
+all: pkg = github.com/icinga/icinga-notifications/internal
```

Comment:
Why are you defining `pkg` this way at this position?

Reply:
Because it's only used in this target. I could probably also set it as a shell variable in just that command.